src/startup.rs

```diff
@@ -5,13 +5,10 @@ use std::panic;
 use std::path::PathBuf;
 
 use crate::common::error_reporting::ErrorReporter;
+use crate::settings::Settings;
 
-const DEFAULT_INITIALIZED_FILE: &str = ".qdrant-initialized";
-
-fn get_init_file_path() -> PathBuf {
-    std::env::var("QDRANT_INIT_FILE_PATH")
-        .map(PathBuf::from)
-        .unwrap_or_else(|_| DEFAULT_INITIALIZED_FILE.into())
+fn get_init_file_path(settings: &Settings) -> PathBuf {
+    PathBuf::from(&settings.init_file_path)
 }
```

Comment:
Let's prefer the `QDRANT_INIT_FILE_PATH` here over the configuration file if set.

That'll ensure backwards compatibility.